### PR TITLE
Cookies for HTTPException.

### PR DESCRIPTION
**Pull request by prostomarkeloff**
## What do these changes do?

Add possbile to add cookies for HTTPException.

## Related issue number

#4277

## Checklist

- [x] I think the code is well written
- [x] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [x] If you provide code modification, please add yourself to `CONTRIBUTORS.txt`
  * The format is &lt;Name&gt; &lt;Surname&gt;.
  * Please keep alphabetical order, the file is sorted by names. 
- [x] Add a new news fragment into the `CHANGES` folder
  * name it `<issue_id>.<type>` for example (588.bugfix)
  * if you don't have an `issue_id` change it to the pr id after creating the pr
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example: "Fix issue with non-ascii contents in doctest text files."

### CODE REVIEW

File: examples/fake_server.py [hunk 1]
```diff
@@ -95,7 +95,7 @@ def __init__(self, *, loop):
 async def main(loop):
     token = "ER34gsSGGS34XCBKd7u"
 
-    fake_facebook = FakeFacebook()
+    fake_facebook = FakeFacebook(loop=loop)
```
Comment (asvetlov) [inline examples/fake_server.py:98]: Please sync with a fresh master; the loop parameter was removed recently.

File: tests/test_web_exceptions.py [hunk 2]
```diff
@@ -9,9 +10,9 @@
 
 
 def test_all_http_exceptions_exported() -> None:
-    assert 'HTTPException' in web.__all__
+    assert "HTTPException" in web.__all__
```
Comment (asvetlov) [inline tests/test_web_exceptions.py:13]: I'm curious why did you replace all quote symbols?
The change is not related to the PR purpose.

File: aiohttp/web_exceptions.py [hunk 3]
```diff
@@ -122,11 +123,80 @@ def __init__(self, *,
...
         self._text = text
         self._headers = real_headers
+        self._cookies: 'SimpleCookie[str]' = SimpleCookie()
         self.args = ()
 
     def __bool__(self) -> bool:
         return True
 
+    @property
+    def cookies(self) -> 'SimpleCookie[str]':
```
Comment (asvetlov) [inline aiohttp/web_exceptions.py:133]: Please not copy this block from `web_response.py` but reuse.
Maybe extraction the common code into some mixin class in `helpers.py` can handle it pretty well.

File: tests/test_web_exceptions.py (hunk 2 above)
Comment (prostomarkeloff) [reply tests/test_web_exceptions.py:13]: Oops, sorry me. It's black.

File: aiohttp/web_exceptions.py (hunk 3 above)
Comment (prostomarkeloff) [reply aiohttp/web_exceptions.py:133]: Sounds right for me, code duplication it's bad way.

Comment (prostomarkeloff) [reply aiohttp/web_exceptions.py:133]: OK, as we can place this code in `helpers.py`? I think new class `CookiesMixin` will be good solution.

Comment (asvetlov) [reply aiohttp/web_exceptions.py:133]: sounds good

File: tests/test_web_exceptions.py (hunk 2 above)
Comment (asvetlov) [reply tests/test_web_exceptions.py:13]: Let's do black reformatting in another PR.
Since 19.10 it can be applied to Python 3.5 compatible code.
The trick is that blackifying should be done on all supported branches at the same time; otherwise backporting becomes virtually impossible.
